source/guide_mongodb.rst

```diff
@@ -157,6 +157,58 @@ Now you can just run ``mongo`` to connect to your MongoDB instance:
 
 You can exit the shell by entering ``exit``.
 
+
+External access (optional)
+--------------------------
+
+If you want to use your Mongodb instance only from your Uberspace account you can skip this part. However, there are cases where you need to connect to your instance from external addresses - e.g. your home computer. You can access your MongoDB instance externally by opening a port and modifying the service config file. Whennever you open a service to the public internet it should be encrypted, we'll take care of that in this example.
+
+.. code-block:: none
+
+ [isabell@stardust ~]$ uberspace port add
+ Port 43120 will be open for TCP and UDP traffic in a few minutes.
+ [isabell@stardust ~]$ 
+
+Modify your ``~/etc/services.d/mongodb.ini`` file. Replace the port number with the port that was opened by the previous command. 
+
+.. code-block:: ini
+ :emphasize-lines: 4,5,8,9
+
+ [program:mongodb]
+ command=mongod
+   --dbpath %(ENV_HOME)s/mongodb
+   --port 43120
+   --bind_ip 0.0.0.0
+   --auth
+   --unixSocketPrefix %(ENV_HOME)s/mongodb
+   --tlsMode requireTLS 
+   --sslPEMKeyFile %(ENV_HOME)s/mongodb/%(ENV_USER)s.uber.space.pem
+ autostart=yes
+ autorestart=yes
+ startsecs=30
+
+
+Create the `.pem` file with the certificate chain:
```

Comment:
The following steps need to be repeated whenever the Let's Encrypt certificate changes (typically every 60 days).